Sources/TuistCore/Models/LintingIssue.swift

```diff
@@ -50,12 +50,9 @@ extension [LintingIssue] {
 
     public func printWarningsIfNeeded() {
         if count == 0 { return }
-
         let warningIssues = filter { $0.severity == .warning }
-        logger.warning("")
         for issue in warningIssues {
-            logger.warning("Warning: \(issue.description)")
+            WarningController.shared.append(warning: issue.description)
```

Comment:
Can we keep using `logger.warning` but change the `LogHandler` we pass during the log bootstrapping?

Reply:
Adjusted! I only did it for the standard logger though because is the one that users use by default. Others preserve the current behaviour.